<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CaptureImg</class>
 <widget class="QWidget" name="CaptureImg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Capture Image</string>
  </property>
  <widget class="QPushButton" name="pushButton_Prev">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>450</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt; Prev</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Next">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>550</y>
     <width>450</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string>Next &gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_CameraOutput">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>960</width>
     <height>512</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_value">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>490</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;000&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_capture_1">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>480</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>capture_1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_capture_2">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>480</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>capture_2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Capture_1">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>10</y>
     <width>384</width>
     <height>216</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0,0,0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Capture_2">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>240</y>
     <width>384</width>
     <height>216</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0,0,0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_value_roll_1">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>590</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;000&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_value_tilt">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>640</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;000&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_value_roll_2">
   <property name="geometry">
    <rect>
     <x>1220</x>
     <y>590</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;000&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>590</y>
     <width>59</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Roll</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_CalculationTilt">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>640</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Tilt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_clear_1">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>530</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_clear_2">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>530</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <zorder>label_CameraOutput</zorder>
  <zorder>pushButton_Prev</zorder>
  <zorder>pushButton_Next</zorder>
  <zorder>textEdit_value</zorder>
  <zorder>pushButton_capture_1</zorder>
  <zorder>pushButton_capture_2</zorder>
  <zorder>label_Capture_1</zorder>
  <zorder>label_Capture_2</zorder>
  <zorder>textEdit_value_roll_1</zorder>
  <zorder>textEdit_value_tilt</zorder>
  <zorder>textEdit_value_roll_2</zorder>
  <zorder>label</zorder>
  <zorder>pushButton_CalculationTilt</zorder>
  <zorder>pushButton_clear_1</zorder>
  <zorder>pushButton_clear_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
